src/app/app-component.jsx

```diff
@@ -35,7 +35,7 @@ module.exports = React.createClass({
     var Page = pageRegistry.getPage(this.state.currentPage);
 
     return (
-      <div id="app-component">{Page({ context: this.props.context })}</div>
+      <div id="app-component">{React.createElement(Page,{ context: this.props.context })}</div>
```

Comment:
So... here's a thing: New React Element Factories and JSX -- https://gist.github.com/sebmarkbage/d7bce729f38730399d28

I've refactored such that I'm not wrapping page components as factories when I insert them into the page registry. This makes it so that I can access static functions of components when accessing them from the registry. I've done this so that I can store the isomorphic "initialize" action creator on the component itself, which is then referenced by the navigate action before navigation finishes. 

Usually this is not necessary. But in this sort of core case, it seemed to make sense.